DAQ_System/Source/LSM303DLHC.cpp

```diff
@@ -0,0 +1,74 @@
+#include "mbed.h"
+#include "LSM303DLHC.h"
+#include "stdio.h"
+ 
+#define MAG_ADDRESS  0x3C
+#define ACC_ADDRESS  0x32
+  
+ 
+LSM303DLHC::LSM303DLHC(PinName sda, PinName scl): _device(sda, scl)
+{
+    _device.frequency(400000);
+//    init();
```

Comment:
Let's remove this comment. I would like `init()` to be separate from the creation of a class object